### ISSUE TITLE
Morale System Reframing

### ISSUE THREAD

**Issue by chichit1044** [CleverRaven/Cataclysm-DDA#67112]
### Is your feature request related to a problem? Please describe.

It seems to be a common opinion that the current morale system isn't what it needs to be. I'd like to suggest that, mechanically, it works pretty well and could be improved with some minor tweaks if instead the fluff and language around it is what's refined.

### Solution you would like.

It is, simply put, not a great idea to try and depict someone's rich internal experience with green and red numbers. More importantly, it's _unsatisfying_. It feels bad to be told your character is experiencing existential crisis, or deep sadness, or what have you, if you don't agree.

The current morale system seems to do just fine on most cases, in that they reflect extremely short term effects on the immediate psyche. They trigger in response to very present things, and wear off comparatively quickly. If you're cold, it makes you unhappy; if you're listening to cool music, you're having fun.

 The stumbling block seems to be trying to integrate larger ideas of worldview, mental illness, trauma, and philosophy into it, which are simply beyond the scope of the system right now. Rather than extensively overhaul the system, I would move that we make it explicitly and only about stuff that will affect the character for a couple hours, axe most stuff that would imply anything about deeper psychological impact, and do some check-based tricks for the middle ground.

For example; right now, Psychopath is a trait that lets you get out of morale penalties for murder, which makes a *lot* of assumptions about what someone is capable of in a crisis. It's not possible to simulate someone's personal journey with trauma in the mechanics. A proposed rework would be to have a version of it, Clear Conscience, trigger after the third or fourth live human someone kills. It doesn't mean that you have made deep peace with murder; it means that you don't get the shakes for ten minutes afterwards anymore. Mood swings becomes Attention Deficit; being very grumpy doesn't make you worse at focusing, but trying to focus and failing makes you grumpy in a red-number way when you're dealing with it. And if the proposed idea of a morale penalty for surviving the cataclysm is added, it should exist for the first two days after the actual cataclysm, tops. Zero morale doesn't reflect nothing being wrong; it reflects your ability to function over the course of the next hour or so, and if you don't have that figured out somehow after the first two days, it's because you're dead.

Basically, the character's overall state of mind shouldn't be diegetic; it should be up to the player. Morale should only reflect very immediate things.

### Describe alternatives you have considered.

The opposite direction; rework morale to reflect overall emotional well-being, with minor stuff like being wet adding only slight variations to effects that last for days or weeks otherwise. I don't have any idea how this could be done in a satisfying way, but that doesn't mean it's not possible.

**Comment by Zireael07** [comment]
Why close?

**Comment by chichit1044** [comment]
> Why close?

Punching above my weight level, so to speak. The morale system is more complex, and older, than I think my input is useful for.

**Comment by Zireael07** [comment]
The fact that the system is very complex does not invalidate your suggestion which should be kept open just to facilitate discussion (closing immediately does NOT)

**Comment by chichit1044** [comment]
I appreciate it, but if there is a useful discussion along these lines to be had, it has probably already been had and I didn't see it. More stuff cluttering the open issues crowds bug reports out of the way.
